docs/index.md

```diff
@@ -36,9 +36,4 @@ in alpha, and other platforms planned.
  Manifests
 * Choose from eight different CNI [Networking](networking.md) providers out-of-the-box
 * Supports upgrading from [kube-up](upgrade_from_kubeup.md)
-* Capability to add containers, as hooks, and files to nodes via a [cluster manifest](cluster_spec.md)
-
-
-## Documentation
-
-[To check out Live documentation](https://kops.sigs.k8s.io/)
```

Comment:
I wonder if we should move this to a comment at the top of this file to tell markdown users they should goto the live docs site?

Reply:
Not sure it is easy to add this without it also being rendered on the live docs site.
Maybe something to look into later?

Reply:
Fair enough. 👍